vaadin-upload-flow-parent/vaadin-upload-flow/src/test/java/com/vaadin/flow/component/upload/DisabledUploadTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2000-2024 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.upload;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DisabledUploadTest {
+    @Test
+    public void preventsStartingUploads() {
```

Comment:
Does it make sense to add an integration test for this? I could not come up with an alternative unit test that avoid using internals.

Reply:
I think it's OK as is, the stream variable is the only entry point to start an upload. Do you feel strongly about this?

Reply:
Not really, LGTM otherwise.